starter-code/configs/session.config.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require('mongoose')
+const session = require("express-session")
+const MongoStore = require("connect-mongo")(session)
+
+module.exports = app => {
+
+    app.use(session({
+        secret: "basic-auth-secret",
+        cookie: { maxAge: 60000 },
```

Comment:
```suggestion
```
Creo que el problema de que la sesión tenga una duración limitada se debe a la configuración del middleware _express-session_, concretamente al valor _maxAge_, te dejo link a la [documentación](http://expressjs.com/en/resources/middleware/cookie-session.html)!!!

_maxAge: a number representing the milliseconds from Date.now() for expiry_